bin/ansible2gdeploy.py

```diff
@@ -24,36 +25,43 @@
 
 def main():
     ap = argparse.ArgumentParser(
-        description="in place update of hosts in gdeploy conf file")
+        description="update of hosts and devices in gdeploy conf file")
     ap.add_argument(
         "-d",
         "--dry-run",
         action="store_true",
-        help="print to stdout instead of in place edit")
-    ap.add_argument("gdeployconf", help="gdeploy config file to edit")
+        help="print to stdout instead of save to file")
+    ap.add_argument(
+        "gdeployconf",
+        nargs="+",
+        help="gdeploy config files to edit")
     ap.add_argument(
         "-i",
         dest="inventory",
         action="store",
         help="ansible inventory (aka hosts) file")
     ap.add_argument(
-        "-o",
-        dest="output",
+        "-p",
```

Comment:
Why do we change command line API? Wouldn't that require update of jobs using this script?

Reply:
Because parameter `-o` is suitable only for one input/output file, so when I changed the script to handle multiple files at once, I have to somehow handle the possibility to update multiple files at once and do not change the original files. The current usage in CI jobs was to save the updated files with prefix *UPDATED_* (handled by loop in bash), so this change is quite suitable for CI usage. It will require just small update in CI jobs.

Reply:
ok